includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1134,18 +1134,29 @@ protected function get_wizard_in_cart_payment_gateways() {
 			'enabled' => $can_stripe && ! ( $should_display_klarna_checkout || $should_display_klarna_payments || $should_display_square ),
 			'featured' => ! ( $should_display_klarna_checkout || $should_display_klarna_payments || $should_display_square ),
 		);
-		$braintree_paypal = array(
-			'name'        => __( 'PayPal by Braintree', 'woocommerce' ),
-			'image'       => WC()->plugin_url() . '/assets/images/paypal-braintree.png',
-			'description' => $paypal_bt_description,
-			'repo-slug'   => 'woocommerce-gateway-paypal-powered-by-braintree',
-		);
 		$ppec_paypal = array(
 			'name'        => __( 'PayPal Express Checkout', 'woocommerce' ),
 			'image'       => WC()->plugin_url() . '/assets/images/paypal.png',
-			'class'       => 'paypal-logo',
```

Comment:
@claudiulodro I added the `paypal-logo` class in this PR to make the logo a bit larger — any reason not to bring it back?

Reply:
There were a bunch of gnarly merge conflicts in that file and it must have gotten removed while resolving them. Thanks for the heads-up. Good catch. I'll add it back.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/commit/4af3f9f12fa88d0919b9b9ff54088d9c27714411